gems/laa_multi_step_forms/app/assets/stylesheets/laa_multi_step_forms/local/_navigation.scss

```diff
@@ -0,0 +1,56 @@
+// Navigation styles
+
+nav {
```

Comment:
what did this affect?

Reply:
I should be able to remove these now - I was originally following CCCD exactly which had some custom css that it used in place of the govuk-frontend styling. I redid it so that it would just use the govuk-frontend styling

Reply:
may a max of 100?